polls/templates/polls/auth_token.html

```diff
@@ -30,12 +32,54 @@ <h3>API Token</h3>
 headers = {
     "Authorization": "Token {{ token.key }}"
     }
-url = "{{ admin_api_url }}"
+url = "{{ admin_api }}"
 response = requests.get(url, headers=headers)
 print(response.json())
 print(response.status_code)
         </code>
       </pre>
+
+      <div class="mt-3">
+        <h4>Annotation API</h4>
+        <p>Fetches all entries of annotation table</p>
+        <pre>
+          <code>
+import requests
+
+headers = {
+    "Authorization": "Token {{ token.key }}"
+    }
+annotation_api = "{{ annotation_api }}"
+response = requests.get(annotation_api, headers=headers)
+print(response.json())
```

Comment:
print statement should be removed after first testing, once it works, there is no need to leave the print statement behind.